Sources/SnowManLogic/View/UI.swift

```diff
@@ -0,0 +1,12 @@
+public class UI: Receivable {
+	public init() {
+	}
+
+	public func getUserInput() -> String {
+		return String(readLine(stripNewline: true) ?? "")
```

Comment:
You're not going to be able to test this, you want `getUserInput` to just wrap `readLine(stripNewLine: true)` move all the logic involved to the view.